.travis.yml

```diff
@@ -20,9 +20,11 @@ matrix:
         homebrew:
           packages:
             - astyle
-            - gcc@8
+            - gcc
```

Comment:
Why do we not explicitly ask for version 8 anymore? Does this not break when homebrew exposes gcc 8.3.1 as `gcc`, as our symlink is pinned to 8.3.0? This implies that we may even want to pin to 8.3.0..

Is this whole mess not resolved by doing `brew link` instead of manually creating a symlink to the Cellar?

Reply:
This was resolved after some offline discussion, in eb08730d277f61e0344c35c9e450fb86fb54503a. By linking to `/usr/local/bin/gcc-8` we are agnostic to minor versions, and by pinning to `gcc@8`, we prevent accidentally bumping a major version.